sdcm/sct_events/file_logger.py

```diff
@@ -74,7 +74,10 @@ def run(self) -> None:
                 self.write_event(event=event, tee=LOGGER.info)
 
     def write_event(self, event: SctEvent, tee: Optional[Callable[[str], Any]] = None) -> None:
-        message = f"{event.formatted_timestamp}: {str(event).strip()}"
+        if event.original_timestamp:
+            message = f"{event.formatted_timestamp} <{event.formatted_timestamp}>: {str(event).strip()}"
```

Comment:
Should be here original_formatted_timestamp?

Reply:
Agree, looks like typo.